packages/mdc-shape/_functions.scss

```diff
@@ -70,39 +72,10 @@
   }
 }
 
-@function mdc-shape-resolve-percentage-for-corner_($component-height, $radius) {
```

Comment:
I moved this function lower in the file verbatim, to group internal functions together.